lib/common.py

```diff
@@ -16,6 +17,15 @@ def vm_image(vm_key):
         url = DEF_VM_URL + url
     return url
 
+def objects_name_prefix():
+    try:
+        from data import OBJECTS_NAME_PREFIX
+        name_prefix = OBJECTS_NAME_PREFIX
+    except ImportError:
+        name_prefix = f"[{getpass.getuser()}]"
+    finally:
```

Comment:
I should have mentioned it earlier, as I had seen it yesterday already: this `finally` statement not only is not necessary, but it's dangerous. It's unlikely, but if an exception was raised that is not ImportError and would prevent name_prefix from being set, we would execute `return name_prefix` on a variable that is not set. Without `finally`, the `return` statement will only be executed if name_prefix is set.